### PR TITLE
Change license from GPL to LGPLv3+

### PR DESCRIPTION
**Pull request by jkonecny12**
This library will have much better benefit from LGPL than GPL. GPL license were inherited from Anaconda project where this project were originated from. However, Anaconda is end user program but Simpleline is general Text UI library. Such a library should be available for anyone so LGPL is much better suited.

The original code from Anaconda was contributed only by Anaconda team members and all of them are/were Red Hat employees. All the new code in simpleline is also contributed only by Anaconda team members who are Red Hat employees. In that case re-licensing is not a problem here.

### CODE REVIEW

File: python-simpleline.spec [hunk 1]
```diff
@@ -10,7 +10,7 @@ Release: 1%{?dist}
 #   cd python-simpleline && make archive
 Source0: https://github.com/rhinstaller/python-%{srcname}/archive/%{srcname}-%{version}.tar.gz
 
-License: GPLv2+
+License: GPLv3+
```
Comment (poncovka) [inline python-simpleline.spec:13]: Shouldn't it be `LGPLv3+`?

Comment (VladimirSlavik) [reply python-simpleline.spec:13]: Haha, indeed. Maybe I should read commit messages :(

**Comment by jkonecny12** [comment]
> Looks good to me, except for the change from COPYING to LICENSE.md ... is that a convention now?

This is now recommended way from GitHub but I think I'll change that back COPYING is more standard for us and GitHub understands that well.

File: python-simpleline.spec (hunk 1 above)
Comment (jkonecny12) [reply python-simpleline.spec:13]: Ups. Thanks a lot for spotting that!

Comment (jkonecny12) [reply python-simpleline.spec]: Fixed.

**Comment by jkonecny12** [comment]
**UPDATED:**
* Fixed license in spec.

**Pull request merged**